src/main/kotlin/no/nav/bidrag/grunnlag/service/GrunnlagspakkeService.kt

```diff
@@ -65,14 +66,19 @@ class GrunnlagspakkeService(
     // Validerer at grunnlagspakke eksisterer
     persistenceService.validerGrunnlagspakke(oppdaterGrunnlagspakkeRequest.grunnlagspakkeId)
 
+/*    if(oppdaterGrunnlagspakkeRequest.gyldigTil != null) {
+      persistenceService.oppdaterGrunnlagspakke(oppdaterGrunnlagspakkeRequest.grunnlagspakkeId,
+      oppdaterGrunnlagspakkeRequest.gyldigTil)
+    }*/
+
```

Comment:
Kan vel kanskje fjernes helt?

Reply:
Will do

Reply:
Fint om du kan se på testene

Reply:
Jeg lar denne stå enn så lenge for å ikke glemme å implementere setting av gyldigTil-dato på et senere tidspunkt